contributing-guides/style-guide.de.md

```diff
@@ -0,0 +1,92 @@
+ [`tldr tldrl`](https://github.com/tldr-pages/tldr/blob/main/pages/common/tldrl.md)
+
+ Viele Clients unterstützen die `--render` Flag zum Anzeigen einer Seite:
+
+ ```
+ tldr --render {{page.md}}
+ ```
+
+ ## Token-Syntax
+
+ Eingaben der Nutzer\*innen sollten die `{{Token}}`-Syntax nutzen,
+ damit `tldr`-Clients sie hervorheben können.
+
+ Die folgenden Regeln sollten für Tokens beachtet werden:
+
+ 1. Kurze und deskriptive Tokens,
+    z. B. `{{source_file}}` oder `{{wallet.txt}}`.
+ 2. Nutze `snake_case` <!--TODO: german wikipedia article for snake_case--> für Tokens, die aus mehreren Wörtern bestehen.
+ 3. Nutze `{{filename}}` statt `{{file_name}}`.
+ 4. Nutze für Referenzen auf Pfade von Dateien oder Ordnern das Format `{{path/to/<Platzhalter>}}`.
+    Beispielsweise `ln -s {{path/to/file}} {{path/to/symlink}}`.
+    Nutze für eine Lücke, in die Dateien und Ordner passen `{{path/to/file_or_directory}}`
+ 5. Folge der `{{path/to/<Platzhalter>}}`-Konvention für alle Pfad-bezogenen Kommandos, außer wenn der
+    Ort der Datei implizit ist.
+ 6. Wenn ein Kommando eine bestimmte Dateiendung erwartet, nutze sie.
+    Beispiel: `unrar x {{compressed.rar}}`.
+    Für eine generelle Dateiendung, nutze `{{.ext}}`, aber **nur**, wenn eine Endung wirklich nötig ist.
+    Beispielsweise, in find.md's Beispiel "Find files by extension" (`find {{root_path}} -name '{{*.ext}}'`)
+    erklärt `{{*.ext}}` den Befehl ohne unnötig spezifisch zu sein;
+    Aber in einem Befehl wie `wc -l {{file}}`, genügt `{{file}}` (ohne Endung).
+ 7. Wenn das Beispiel mit einem konkreten Wert klarer ist, nutze einen Beispielwert.
+    Nutze beispielsweise `iostat {{2}}` statt `iostat {{interval_in_secs}}`.
+ 8. Wenn ein Kommando irreversible Änderungen am Dateisystem oder Geräten vornimmt, schreibe jedes Beispiel so, dass es nicht blind copy-pastet werden kann.
+    Schreibe beispielsweise statt `ddrescue --force --no-scrape /dev/sda /dev/sdb` `ddrescue --force --no-scrape {{/dev/sdX}} {{/dev/sdY}}` und nutze den `{{/dev/sdXY}}`-Platzhalter für *block devices* statt `/dev/sda1`.
+
+ Generell sollten Tokens es so intuitiv wie möglich machen,
+ herauszufinden, wie das Kommando funktioniert und sie mit Werten auszufüllen.
+
+ Technische Begriffe in der Beschreibung sollten die `Backtick`-Syntax (\`) benutzen.
+ Nutze Backticks für Folgendes:
+
+ 1. Pfade, wie `package.json`, `/etc/package.json`.
+ 2. Dateiendungen, wie `.dll`.
+ 3. Kommandos, wie `ls`.
+
+ ## Serial Comma
+
+ Nutze für eine Liste von 3 oder mehr Elementen ein [serial comma](https://en.wikipedia.org/wiki/Serial_comma), um Mehrdeutigkeiten zu verhindern.
+
+ > Delete the Git branches, tags and remotes.
+
+ Das obige Beispiel nutzt kein serial comma und ist somit mehrdeutig:
+ * Lösche die Git Branches namens `tags` und `remotes`.
+ * Lösche die Git Branches und die Git Tags und die Git Remotes.
+
+ Dies kann durch ein Komma vor "and" oder "or" gelöst werden.
+
+ > Delete the Git branches, tags, and remotes.
```

Comment:
Something like the `Serial comma` dosn't exist in German as far as I know.
```suggestion
```

Reply:
Well, you can say `Branches, Tags, und Remotes`, but also: How does that matter? It does in English and it is important to `tldr`, so German contributors should know about it.